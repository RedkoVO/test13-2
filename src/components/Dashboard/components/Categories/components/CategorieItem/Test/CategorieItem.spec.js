import * as React from 'react'
import renderer from 'react-test-renderer'
import { MemoryRouter } from 'react-router-dom'

import CategoriesItem from '../'

const data = {
  id: 1,
  title: 'Most popular',
  name: '4Clover',
  img: 'CloverLogo1',
  bundle: '4clover'
}

it('renders correctly', () => {
  const tree = renderer
    .create(
      <MemoryRouter keyLength={0}>
        <CategoriesItem game={data} />
      </MemoryRouter>
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
