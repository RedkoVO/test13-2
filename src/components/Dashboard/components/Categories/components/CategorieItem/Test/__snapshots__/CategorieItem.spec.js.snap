// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="CategorieItem-root-1"
>
  <a
    className="CategorieItem-gameImg-2"
    href="/game/4clover"
    onClick={[Function]}
  >
    <img
      alt=""
      className="CategorieItem-gameImgLink-3"
      src="CloverLogo1"
    />
  </a>
  <a
    className="CategorieItem-gameLink-4"
    href="/game/4clover"
    onClick={[Function]}
  >
    Most popular
    : 
    <br />
     
    <span>
      4Clover
    </span>
  </a>
</div>
`;
